README.cartwheel

```diff
@@ -4,6 +4,23 @@ Intel developer staging area for upstream patch contributions to [FFmpeg](https:
 
 The upstream project is tracked as a submodule in this repo.
 
+## upstream process
+To improve upstream patch quality and reduce patchset rebase efforts, we use the following process
+1. Send PR1 to https://github.com/intel-media-ci/ffmpeg, add module maintainers to Reviewers
+1. Got approval from one maintainer
+1. Wait for 1 more workday to see any objections. If it's an urgent issue, please ask another reviewer + 1 for you.
```

Comment:
Did you intend to use 1/2/3... before each item?

Reply:
yes, it's markdown syntax.
see https://github.com/xuguangxin/cartwheel-ffmpeg/tree/process for final result.

Reply:
OK, I see. Then LGTM.